scripts/tmux.sh

```diff
@@ -68,14 +72,15 @@ run_mapi() {
   tmux rename-window -t $SESSION:$window "api"
   tmux send-keys -t $SESSION:$window "docker compose up --build -d" C-m
   tmux send-keys -t $SESSION:$window "# Make sure you wait for everything to come up" C-m # Wait for everything to come up
-  tmux send-keys -t $SESSION:$window "mapi run ${WORKSPACE}/mayhem-demo/api 1m https://localhost:8443/openapi.json --url https://localhost:8443 --sarif mapi.sarif --html mapi.html --interactive --basic-auth 'me@me.com:123456' --ignore-rule internal-server-error --experimental-rules"
+  tmux send-keys -t $SESSION:$window "mapi run ${WORKSPACE}/${PROJECT}/api 1m https://localhost:8443/openapi.json --url https://localhost:8443 --sarif mapi.sarif --html mapi.html --interactive --basic-auth 'me@me.com:123456' --ignore-rule internal-server-error --experimental-rules"
 }
 
 run_mapi_discover() {
   window=1
   tmux new-window -t $SESSION:$window -n "discover"
-  tmux send-keys -t $SESSION:$window " mapi discover --domains demo-api.mayhem.security --endpoints-file ./scripts/endpoints.txt" C-m
+  # tmux send-keys -t $SESSION:$window " mapi discover --domains demo-api.mayhem.security --endpoints-file ./scripts/endpoints.txt" C-m
```

Comment:
Should this be removed instead of just being commented out?

Reply:
We may want to add it back in later (someone else authored this line), so I'm commenting it for now.